src/components/profile/Profile.js

```diff
@@ -33,6 +37,7 @@ const ProfileWrapper = props => {
     if (isEqual(profile, {})) {
       updateProfile()
     }
+    userStorage.onProfile(handleChangeProfile)
```

Comment:
This should probably use gundb .once and not .on since any future change to profile will envoke this callback.
We just want to make sure we have the profile the first time.

Reply:
so we need receive once and then unsubscribe. ok